src/main/java/bio/terra/cli/app/CommandRunner.java

```diff
@@ -123,7 +123,8 @@ private Map<String, String> buildMapOfTerraReferences() {
         .forEach(
             resource -> {
               if (Resource.Type.DATA_COLLECTION != resource.getResourceType()) {
-                terraReferences.put("TERRA_" + resource.getName(), resource.resolve());
+                String envVariable = convertToEnvironmentVariable("TERRA_" + resource.getName());
```

Comment:
optional nit: move the `"TERRA_"+` inside `convertToEnvironmentVariable`?

Reply:
Done